### DISCUSSION TITLE
acceptance-test-vs-e2e-test/

### DISCUSSION THREAD

**Discussion by giscus** [Hyeon9mak/Hyeon9mak.github.io#172] [Blog comments]
# acceptance-test-vs-e2e-test/

🪧 summary  

https://hyeon9mak.github.io/acceptance-test-vs-e2e-test/

**Comment by JuTiger-Lee** [comment]
인수 테스트란 사용자 관점에서 테스트하는거고 E2E 테스트는 개발자 입장에서 기능 테스트를 하는거군요. 정말 관점 차이네요 ㅋㅋㅋㅋ

좋은 글 잘 읽었습니다 감사합니다.

**Comment by junghoon-vans** [comment]
궁금했던 부분인데 덕분에 말끔히 해결되었네요 👍
